tests/integration/test_vid_club.py

```diff
@@ -64,27 +101,80 @@ def test_get_response_wrong_source():
 @pytest.mark.parametrize("source", ["jobs", "company", "product", "survey"])
 @pytest.mark.proper
 def test_get_response_sources(mock_api_response, source):
-    vc = VidClub()
     query = vc.get_response(source=source, to_date="2023-03-24", from_date="2023-03-24")
 
     assert isinstance(query, pd.DataFrame)
+    mock_api_response.assert_called_once()
 
 
 @pytest.mark.proper
 def test_get_response_wrong_date():
     with pytest.raises(
-        ValidationError, match=r"from_date cannot be earlier than 2023-03-22!!"
+        ValidationError, match=r"from_date cannot be earlier than 2022-03-22"
     ):
-        vc = VidClub()
         query = vc.get_response(source="jobs", from_date="2021-05-09")
 
 
 @pytest.mark.proper
 def test_get_response_wrong_date_range():
     with pytest.raises(
-        ValidationError, match=r"to_date cannot be earlier than from_date!"
+        ValidationError, match=r"to_date cannot be earlier than from_date"
     ):
-        vc = VidClub()
         query = vc.get_response(
             source="jobs", to_date="2022-05-04", from_date="2022-05-05"
         )
+
+
+@pytest.mark.total_load
+def test_total_load_for_the_same_dates():
+    from_date = "2022-04-01"
+    to_date = "2022-04-01"
+    days_interval = 10
+    source = "jobs"
+    df = vc.total_load(
+        from_date=from_date, to_date=to_date, days_interval=days_interval, source=source
+    )
+
+    assert isinstance(df, pd.DataFrame)
+
+
+@pytest.mark.total_load
+def test_total_load_for_intervals():
+    from_date = "2022-04-01"
+    to_date = "2022-04-12"
+    days_interval = 2
+    source = "jobs"
+
+    date_object = datetime.strptime(from_date, "%Y-%m-%d") + timedelta(
+        days=days_interval
+    )
+    one_interval = date_object.strftime("%Y-%m-%d")
+
+    df = vc.total_load(
+        from_date=from_date, to_date=to_date, days_interval=days_interval, source=source
+    )
+    df_one_interval = vc.total_load(
+        from_date=from_date,
+        to_date=one_interval,
+        days_interval=days_interval,
+        source=source,
+    )
+
+    assert isinstance(df, pd.DataFrame)
+    assert len(df) > len(df_one_interval)
+
+
+@pytest.mark.total_load
+def test_drop_duplicates():
+    from_date = "2022-04-01"
+    to_date = "2022-04-12"
+    days_interval = 2
+    source = "jobs"
+
+    df = vc.total_load(
+        from_date=from_date, to_date=to_date, days_interval=days_interval, source=source
+    )
+    dups_mask = df.duplicated()
+    df_check = df[dups_mask]
+
+    assert len(df_check) == 0
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing